stories/atoms/components/list/List.stories.mdx

```diff
@@ -0,0 +1,284 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs'
+import { List } from 'ui/atoms/list/List'
+import { ListItem } from 'ui/atoms/listItem/ListItem'
+import { Typography } from 'ui/atoms/typography/Typography'
+import { ProgressBar } from 'ui/atoms/progressBar/ProgressBar'
+import { Icon } from 'ui/atoms/icon/Icon'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+
+<Meta
+  title="Atoms/List"
+  component={List}
+  parameters={{
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      source: {
+        type: 'code'
+      }
+    }
+  }}
+  decorators={[
+    Story => (
+      <ThemeProvider>
+        <div className="component-story-main">
+          <ThemeSwitcher />
+          <div
+            style={{
+              display: 'grid',
+              gap: '25px'
+            }}
+          >
+            <Story />
+          </div>
+        </div>
+      </ThemeProvider>
+    )
+  ]}
+/>
+
+# List
+
+> A primary UI component to display a list of items.
+
+[![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
+
+## Description
+
+The `List` allows to display a list of elements and manage them.
+
+## Overview
+
+<Story
+  name="Overview"
+  argTypes={{
+    items: { control: false },
+    leftIcon: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    },
+    rightIcon: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    }
+  }}
+  args={{
+    items: [
+      {
+        label: 'Item 1',
+        description: 'Description 1'
+      },
+      {
+        label: 'Item 2',
+        description: 'Description 2'
+      },
+      {
+        label: 'Item 3',
+        description: 'Description 3'
+      },
+      {
+        label: 'Item 4',
+        description: 'Description 4'
+      },
+      {
+        label: 'Item 5',
+        description: 'Description 5'
+      }
+    ],
+    leftIcon: <Icon name="wallet" />,
+    rightIcon: <Icon name="close" />
+  }}
+>
+  {args => <List {...args} />}
+</Story>
+
+## Properties
+
+<ArgsTable story="Overview" />
+
+## ListItem
+
+The property `items` provides the list of items to display.
+
+<Story
+  name="ListItem"
+  argTypes={{
+    description: {
+      table: {
+        type: { summary: 'string | JSX.Element' }
+      }
+    },
+    leftIcon: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    },
+    rightIcon: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    },
+    items: {
+      table: {
+        disable: true
+      }
+    }
+  }}
+  args={{
+    label: 'Item',
+    description: 'Description',
```

Comment:
I would have liked to have the control of the label and the description in the table of arguments, but a priori it does not work...
![image](https://user-images.githubusercontent.com/47554752/185090294-1cf0c391-ba76-47ca-8580-d94d4b377b5e.png)

Reply:
Yes it doesn't work because it is the properties table of the `ListItem` component (not the `List`). It is generated by `<ArgsTable of={ListItem} />` and not some thing like : `<ArgsTable story="StoryName" />`
So we can't control the properties here.

Reply:
@JoeyDjr 
Thanks for your explanation, I understand better now 👌
According to what you said, I'll suggest to write a dedicated story for the `ListItem` component and make the link between the two stories.